services/web-app/components/dashboard/dashboard.module.scss

```diff
@@ -110,7 +110,7 @@
   top: convert.rem(-3px);
   width: 1rem;
   height: 1rem;
-  margin-right: spacing.$spacing-03;
+  margin-right: spacing.$spacing-02;
```

Comment:
Did you confirm this with design? It doesn't match the redline, but there isn't a specific spacer noted in Figma either. 

https://www.figma.com/file/vtmHnw9ACgnksFg7ZvEsZC/RL_Asset_ComponentDetails?node-id=0%3A1

<img width="321" alt="Screen Shot 2022-08-30 at 12 01 43 PM" src="https://user-images.githubusercontent.com/2753488/187496838-7b2e3bcf-2de0-4af3-892f-955b8fa3d1c1.png">
<img width="201" alt="Screen Shot 2022-08-30 at 12 01 57 PM" src="https://user-images.githubusercontent.com/2753488/187496846-27fda52f-4ae1-4bb7-8862-475ad422a17a.png">

Reply:
@kingtraceyj

Reply:
@kingtraceyj @jeanservaas 

8px spacing:
<img width="682" alt="image" src="https://user-images.githubusercontent.com/1691245/187497591-c733cfe5-21cd-4773-8584-f70db0c3ca23.png">

4px spacing:
<img width="358" alt="image" src="https://user-images.githubusercontent.com/1691245/187497696-c015a99e-6c75-4c42-b5f4-6ade3b8329b0.png">

This PR updates both to be 4px spacing, so the framework icon spacing matches the status icon, which was in there first.

Is that okay?

Reply:
This PR standardizes on using `$spacing-02` between icon and text in these two places.